src/test_heap.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+"""Testing module for the Heap classes."""
+
+from heap import Heap
+import pytest
+import random
+
+
+# ------------------Heap Initialization Tests--------------------
+# [x] Test default initialization of a heap is a heap.
+# [x] Test the inital size of a heap is 0.
+# [x] Test the default initializtion is a max heap.
+# [x] Test the override to a min heap.
+# [x] Test if invalid input into min/max heap results in an error.
+
+def test_heap_initalization():
+    """Test default initialization of a heap is a heap."""
+    heap = Heap()
+    assert isinstance(heap, Heap)
```

Comment:
These types of tests are not particularly meaningful.  If initializing a `Heap` resulted in anything that wasn't actually a `Heap`, there would probably be something wrong with Python itself (or you'd have built some sort of mutant Python class).